git/README.md

```diff
@@ -7,6 +7,15 @@ tagline: |
 
 To update or switch versions, run `webi git@stable` (or `@v2.30`, `@beta`, etc).
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
```

Comment:
Also:

```text
~/.gitconfig
~/.local/opt/git # Windows
/Library/Developer/CommandLineTools # macOS
```

Reply:
You're right, this one slipped, sorry. Updated as described.